src/cljs/main/broadfcui/page/workspace/monitor/common.cljs

```diff
@@ -74,16 +74,17 @@
 (defn icon-for-sub-status [wf-statuses]
   (cond
     (contains? wf-statuses :Failed) failure-icon
+    (contains? wf-statuses :Aborted) failure-icon
     (contains? wf-statuses :Succeeded) success-icon
-    :else (do (utils/log "Unknown submission status")
+    :else (do (utils/log "Unknown submission status: " wf-statuses)
```

Comment:
Not your code I know, but I can't imagine these log statements add any value, since we have no way to collect logs from users' browsers. I'm inclined to continue the practice of "don't commit log statements" to keep the debugging noise down.